docs/source/protocol_v3.rst

```diff
@@ -1,51 +1,437 @@
-        <val> (bit) (imdwait: true or false. if true, the simulation will wait for the client to send a go signal before starting)
-        <val> (bit) (imdterm: true or false. if true, the client can terminate the simulation)
-        <val> (bit) (wrapped positions: true or false. if positions rate is 0, this is a placeholder value)
+   Header:
+      <val> (int32) Header type
+      <val> (dtype) Data slot description
 
-        <val> (uint32) (energies rate: number of steps that elapse between steps that contain energy data. 0 means never)
-        <val> (uint32) (dimensions rate: number of steps that elapse between steps that contain dimension data. 0 means never)
-        <val> (uint32) (positions rate: number of steps that elapse between steps that contain position data. 0 means never)
-        <val> (uint32) (velocities rate: number of steps that elapse between steps that contain velocity data. 0 means never)
-        <val> (uint32) (forces rate: number of steps that elapse between steps that contain force data. 0 means never)
+Some header packets sent by the simulation engine have associated body packets that contain additional data, 
+like atomic coordinates. These body packets vary in length and composition,
+but their bytes are always encoded in the native endianness of the machine running the simulation engine.
 
-   "wrapped positions" will be a new ``.mdp`` setting which specifies whether the atoms' positions
-   should be adjusted to fit within the simulation box before sending. This is useful for visualization purposes.
+This table lists all header types available in IMDv3. Below, each header type 
+and its associated body packet (if present) is described in detail.
 
-3. The server should wait longer than 1 second (possibly up to 60s) for the go signal so that the client 
-   has plenty of time to allocate memory buffers based on the endianness and information on included data types 
-   it received in the handshake packet.
+.. list-table::
+   :widths: 10 10
+   :header-rows: 1
 
-4. In the simulation loop, the server will send the client data in this order (if the configuration says to send it)
-    
-    i. Energy data (IMD_ENERGIES) unchanged
-    
-    ii. Dimension data (IMD_BOX) in triclinic vectors
+   * - Header type
+     - 32-bit integer enum value
+   * - Disconnect
+     - 0
+   * - Energies
+     - 1
+   * - Coordinates
+     - 2
+   * - Go
+     - 3 
+   * - Handshake
+     - 4
+   * - Kill
+     - 5
+   * - MD Communication
+     - 6
+   * - Pause
+     - 7 
+   * - Transmission rate
+     - 8
+   * - IO Error
+     - 9
+   * - Session info
+     - 10
+   * - Resume
+     - 11
+   * - Time
+     - 12
+   * - Box
+     - 13
+   * - Velocities
+     - 14
+   * - Forces
+     - 15
 
-    iii. Position data (IMD_FCOORDS) unchanged except box adjustments (see 5)
-    
-    iv. Velocity data (IMD_VELS) in the same manner as positions
+Disconnect
```

Comment:
Add anchors in reST in front of each package so that one can easily reference with links.